tests/.env.test

```
@@ -1,18 +1,15 @@
+NEXT_PUBLIC_SITE_URL=https://develop.jumper.xyz
 NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID=G-T6ZDXLLXB5
-# NEXT_PUBLIC_LIFI_BACKEND_URL=https://api-develop.jumper.xyz/pipeline
-NEXT_PUBLIC_LIFI_BACKEND_URL=https://api-develop.jumper.exchange/pipeline
+NEXT_PUBLIC_LIFI_BACKEND_URL=https://api-develop.jumper.xyz/pipeline
 NEXT_PUBLIC_WIDGET_INTEGRATOR=dev.jumper.exchange
 NEXT_PUBLIC_WIDGET_INTEGRATOR_REFUEL=dev.jumper.exchange.gas
 NEXT_PUBLIC_WIDGET_INTEGRATOR_BLOG=dev.jumper.exchange.blog
 NEXT_PUBLIC_WIDGET_INTEGRATOR_EARN=jumper.exchange.earndev
 NEXT_PUBLIC_ADDRESSABLE_TID=0d48165d8b544e029b816f54c3371ae8
 NEXT_PUBLIC_CUSTOM_RPCS={ "59140": ["https://consensys-zkevm-goerli-prealpha.infura.io/v3/faf4bc4ea7344e5da5e56c55de087480"], "1151111081099710": ["https://chaotic-restless-putty.solana-mainnet.quiknode.pro/", "https://dacey-pp61jd-fast-mainnet.helius-rpc.com/"] }
 NEXT_PUBLIC_DKEY=AmE4rjRHH0aLksP_-Nym-DCL_5qOHBYR77m7svbGyHm5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_DKEY key should go before the NEXT_PUBLIC_ENVIRONMENT key

(UnorderedKey)